app/controllers/maily/emails_controller.rb

```diff
@@ -43,15 +43,15 @@ def edit
     def update
       @maily_email.update_template(params[:body], params[:part])
 
-      redirect_to maily_email_path(mailer: params[:mailer], email: params[:email], part: params[:part]), notice: 'Template updated!'
+      redirect_to maily_email_path(mailer: params[:mailer], email: params[:email], version: params[:version], part: params[:part]), notice: 'Template updated!'
```

Comment:
this is just a refactor proposal, but instead of carrying the same params everywhere, we can write a helper, ala `maily_params`, which returns the "present" params as a hash